CRM/Civixero/Contact.php

```diff
@@ -256,13 +268,13 @@ public function push(array $params, int $limit = 10): bool {
           ->setValues($record)
           ->addValue('accounts_needs_update', FALSE)
           ->execute();
+        $contactIDsPushed[] = $record['contact_id'];
       }
-      catch (CRM_Civixero_Exception_XeroThrottle $e) {
```

Comment:
Don't think this is required now  it's moved to API wrapper

Reply:
What about when it fails mid loop